packages/oas-to-har/__tests__/lib/remove-undefined-objects.test.js

```diff
@@ -4,10 +4,21 @@ test('should remove empty objects with only undefined properties', () => {
   expect(removeUndefinedObjects({ a: { b: undefined, c: { d: undefined } } })).toBeUndefined();
 });
 
-test('should not throw on arrays of primitives', () => {
```

Comment:
Did you remove this test? Is it no longer required? Or it now returns undefined completely instead of `[null]`?

Reply:
Removed it because `[null]` should return `undefined` now.